### ISSUE TITLE
[12.0] hr_holidays: fr.po is not in sync with hr_holidays.pot

### ISSUE THREAD

**Issue by ypapouin** [odoo/odoo#71082]
https://www.transifex.com/odoo/odoo-12/translate/#fr/hr_holidays/
I'm not able to translate some missing terms on transifex like:

```
#. module: hr_holidays
#: code:addons/hr_holidays/models/hr_leave.py:666
#, python-format
msgid "%s on Time Off : %.2f day(s)"
msgstr ""

#. module: hr_holidays
#: code:addons/hr_holidays/models/hr_leave.py:664
#, python-format
msgid "%s on Time Off : %.2f hour(s)"
msgstr ""
```

`fr.po` should be msgmerged with `hr_holidays.pot`

**Comment by Yenthe666** [comment]
@mart-e do you need to push some updates to Transifex? 🤔

**Comment by mart-e** [comment]
Hi,
It was an error in the .pot file. It should be fixed at #71101
Transifex will detect the changes and update the content automatically (let me know if still not there in a few days).
